src/index.ts

```diff
@@ -5,31 +5,35 @@ import { AssetMethods } from "./library/AssetMethods";
 import { Synthetics } from "./protocol/Synthetics";
 import Assets from "../src/assets.json";
 import { Utils } from "./library/Utils"
+import { Asset } from "./library/Asset";
 require('dotenv').config();
 
+type supportNetwork = keyof typeof Assets
+type assetGroup = keyof typeof Assets['mainnet'] | keyof typeof Assets['kovan']
```

Comment:
I don't love this since it's clear to me you guys want to keep the sdk fairly dynamic. 
@byterose can you think of an alternative to get a type alias of the keys under the networks?

Reply:
Have a look at the new types in ```./src/types``` : ]